core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -3158,6 +3158,28 @@ private static int unsetLocalDictForComplexColumns(List<ColumnSchema> allColumns
     return columnLocalDictGenMap;
   }
 
+
+  public static void setPageBloomColumnsToWrapperSchema(List<ColumnSchema> columns,
+        Map<String, String> mainTableProperties) {
+    if (!mainTableProperties.containsKey(CarbonCommonConstants.PAGE_BLOOM_INCLUDE)) {
```

Comment:
here checking contains and down calling .get method, instead just call get method and validate for null